### PR TITLE
Made the displayed videos scrollable

### PR DESCRIPTION
**Pull request by Jasperkroes**
in order to keep the progress bar from disappearing off off the screen.

### CODE REVIEW

File: src/main/java/gui/panes/MenuPane.java [hunk 1]
```diff
@@ -301,16 +301,17 @@ private void setCameraStatus() {
...
+        mediaPane.getMediaPlayerPane().setContent(stackPane);
     }
 
     /**
      * Close the stream(s) and reset the application.
      */
     public void endStream() {
         timerManager.stopTimer();
-        mediaPane.getMediaPlayerPane().getChildren().clear();
+//        mediaPane.getMediaPlayerPane().getChildren().clear();
```
Comment (egedebruin) [inline src/main/java/gui/panes/MenuPane.java]: ahum...

Comment (Jasperkroes) [reply src/main/java/gui/panes/MenuPane.java]: ¯\_(ツ)_/¯¯\_(ツ)_/¯¯\_(ツ)_/¯¯\_(ツ)_/¯¯\_(ツ)_/¯¯\_(ツ)_/¯¯\_(ツ)_/¯¯\_(ツ)_/¯¯\_(ツ)_/¯¯\_(ツ)_/¯

**Review by lisv — APPROVED** [review]
It works!

**Review by egedebruin — APPROVED** [review]
I checked and it works fine good work jasper kutgewoon

**Comment by egedebruin** [comment]
kutgw

**Pull request merged**